testnet/configure_node.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+NET_NODE_KEY=$(./etc/nomos/register_node.sh)
+CONSENSUS_NODE_KEY=$NET_NODE_KEY
```

Comment:
What's the point of this variable?

Reply:
These variables are used by the node, added a comment.

Reply:
No, the `CONSENSUS_NODE_KEY` variable is not used anywhere.

Reply:
And if it's supposed to be inherited by a child process then it needs an `export` for that to happen.

Reply:
Thanks a ton, default values prevented nodes from failing.